src/SonarScanner.MSBuild.PreProcessor/WebClientDownloader.cs

```diff
@@ -32,31 +32,14 @@ namespace SonarScanner.MSBuild.PreProcessor
 {
     public class WebClientDownloader : IDownloader
     {
-        // This is a temporary solution until we upgrade to .net framework 4.7.
-        private const SecurityProtocolType SystemDefault = 0;
-
         private readonly ILogger logger;
         private readonly HttpClient client;
 
         public WebClientDownloader(string userName, string password, ILogger logger, string clientCertPath = null, string clientCertPassword = null)
-            : this(userName, password, logger, new SecurityProtocolHandler(), clientCertPath, clientCertPassword) { }
-
-        internal /* for testing */ WebClientDownloader(string userName,
-                                                       string password,
-                                                       ILogger logger,
-                                                       ISecurityProtocolHandler securityProtocolHandler,
-                                                       string clientCertPath = null,
-                                                       string clientCertPassword = null)
         {
             this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
             password = password ?? string.Empty;
```

Comment:
I don't remember the version of the language that we use, if it allows for
```suggestion
            password ??= string.Empty;
```

Reply:
not blocking the PR for this though :)